jac-uk/admin
# #1505 Restrict IA upload to Word files only

## What's included?
Users should be able to upload only .doc/.docx files for Independent Assessments on Exercise Downloads page in Admin.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

- Go to existing exercise, unlock it if needed. 
- Go to Exercise Downloads page.
- Upload Independent Assessors document (please note, you will only be able to see .doc/.docx files in the upload window).
- Upload any other document (you should be able to upload .txt, .pdf, .doc/.docx, .odt, .fodt files).

## Risk - how likely is this to impact other areas?

🟠 Medium risk - this does change code that is shared with other areas

## Additional context

![Upload2](https://user-images.githubusercontent.com/40855898/137474843-d7eab0ed-efc3-4417-81d4-3b2f890c27c7.png)


![Upload1](https://user-images.githubusercontent.com/40855898/137474876-302dc2a2-2801-4ad7-98bb-79daa2be334d.png)


![Screenshot 2021-10-15 at 10 46 15](https://user-images.githubusercontent.com/40855898/137474923-0949911d-05ff-4e7c-93e0-9d738ff7238d.png)


![Screenshot 2021-10-15 at 10 45 24](https://user-images.githubusercontent.com/40855898/137475083-fa1bb8b3-79e8-471f-95b0-e002b7284e52.png)

---

PREVIEW:DEVELOP
